stan/math/rev/scal/fun/frexp.hpp

```diff
@@ -0,0 +1,50 @@
+#ifndef STAN_MATH_REV_SCAL_FUN_FREXP_HPP
+#define STAN_MATH_REV_SCAL_FUN_FREXP_HPP
+
+#include <stan/math/rev/core.hpp>
+#include <cmath>
+#include <limits>
+
+namespace stan {
+  namespace math {
+
+    /**
+	 * Variable implementation of frexp. The derivative of frexp
+	 * is 0, because frexp returns a discontinuous object.
```

Comment:
@charlesm93, is the derivative 0 or NaN? In the `chain()` method, it's using NaN.